### DISCUSSION TITLE
Download posters in media folder instead of assets

### DISCUSSION THREAD

**Discussion by Droppisalt** [meisnate12/Plex-Meta-Manager#1415] [Q&A]
Hello! Is there any way to use `download_url_assets: true` to download posters within the media's folder

My media is located in:
- .../media/movies/anime
- .../media/movies/french
- .../media/movies/general
- .../media/tv/anime
- .../media/tv/french
- .../media/tv/general

So let's say for Loki (2023), it would download posters for the show and seasons (already in a PMM metadata file) to: `/media/tv/general/Loki (2021) [tvdbId-362472]/` and `/media/tv/general/Loki (2021) [tvdbId-362472]/Season 01`

I'm trying to do this because I want to make Jellyfin update it's metadata with those posters.

**Comment by chazlarson** [comment]
PMM doesn't require or use access to the media file system, it's designed to run remote and communicates with Plex via the API.  For that reason it can't write into those folders.

I have some scripts that can download current artwork into your media directories or a parallel matching filesystem that you could merge, but it's unlikely this will be added to PMM.

**Reply by Droppisalt** [reply]
Makes sense. How do your scripts work? With PMM or separate?

**Reply by chazlarson** [reply]
Separate.
